### ISSUE TITLE
Feature request: update gauge's `min` value

### ISSUE THREAD

**Issue by ismirsehregal** [rstudio/flexdashboard#254]
Please check the following example:

```
library(shiny)
library(flexdashboard)

ui <- fluidPage(
    titlePanel("Gauge Test"),
    sidebarLayout(
        sidebarPanel(
            sliderInput("range",
                        "Min / Max:",
                        min = 1,
                        max = 50,
                        value = c(20, 30))
        ),
        mainPanel(
            gaugeOutput(outputId = "myGauge", width = "30%", height = "200px")
        )
    )
)

server <- function(input, output) {
    output$myGauge <- renderGauge({
        gauge(value = mean(input$range), min = min(input$range), max = max(input$range))
})}

shinyApp(ui, server)
```
The `min` value doesn't change, it freezes with the initial `20`:
![screen](https://user-images.githubusercontent.com/36849480/74930142-404eb800-53dd-11ea-85c1-4e932802d27d.gif)

PS: using the latest CRAN versions.

The issue was first seen [here](https://stackoverflow.com/questions/60253611/shiny-flexdasboard-reactive-gauge-not-updating/).

**Comment by cderv** [comment]
Hi, 

I think this is not currently supported by the js library behind `gauge` function. 
Here is how the gauge is updated
https://github.com/rstudio/flexdashboard/blob/404cdac92d4bc1934e3b3a2c514daba0601d050a/inst/htmlwidgets/gauge.js#L77
There is no min value possible in the `.refresh` function from upstream JustGage js library
https://github.com/rstudio/flexdashboard/blob/404cdac92d4bc1934e3b3a2c514daba0601d050a/inst/htmlwidgets/lib/justgage/justgage.js#L860

So currently, the min of the gauge is fixed at creation. 

I guess this would not be so difficult to support as a new feature. Upstream `JustGage` added support for this only 7 month ago (https://github.com/toorshia/justgage/commit/b51840c507e155d3982aa0a17b5b8e02b5687145). It must just be done with care for backward compatibility. Either by updating upstream or modifying the customisation here. 

Hope it helps.

**Comment by ismirsehregal** [comment]
Ah ok, thanks for your investigation! - Leaving this open as a feature request.

**Comment by vickibendus** [comment]
I found a further bug with the gauge bar not showing when 2 inputs are needed for the output value. The bar only shows if it happens one of the min or max values.

```
library(shiny)
library(shinydashboard)
library(flexdashboard)
library(dplyr)

df <- structure(list(Name = structure(c(1L, 3L, 4L, 2L, 1L, 3L, 4L, 
                                        2L, 1L, 3L, 4L, 2L, 1L, 3L, 4L, 2L), .Label = c("Bob", "Fred", 
                                                                                        "Joe", "Ted"), class = "factor"), Date = structure(c(18294, 18294, 
                                                                                                                                             18294, 18294, 17946, 17946, 17946, 17946, 18294, 18294, 18294, 
                                                                                                                                             18294, 17946, 17946, 17946, 17946), class = "Date"), Variable = structure(c(1L, 
                                                                                                                                                                                                                         1L, 1L, 1L, 1L, 1L, 1L, 1L, 2L, 2L, 2L, 2L, 2L, 2L, 2L, 2L), .Label = c("Height", 
                                                                                                                                                                                                                                                                                                 "Weight"), class = "factor"), Value = c(87L, 81L, 99L, 94L, 85L, 
                                                                                                                                                                                                                                                                                                                                         89L, 83L, 95L, 86L, 93L, 90L, 98L, 92L, 99L, 100L, 91L)), row.names = c(NA, 
                                                                                                                                                                                                                                                                                                                                                                                                                 -16L), class = "data.frame")

ui = shinyUI(
    fluidPage(
        sidebarLayout(
            sidebarPanel(selectInput("Variable","Variable", choices = unique(df$Variable))),
            mainPanel(
                box(selectInput("Name", label = "Name", choices = unique(df$Name)), gaugeOutput("Gauge"))
        )
    )))

server <- shinyServer(function(input, output){
    
    output$Gauge <- renderGauge({
      
        t <- df %>%
            filter(Name == input$Name & Variable == input$Variable)
        
        f <- df %>%
            filter(Name == input$Name & Date == c(max(df$Date)) & Variable == input$Variable)
        
        gauge(f$Value, min = min(t$Value), max = max(t$Value))
    })
})

shinyApp(ui = ui, server = server)
```

**Comment by cpsievert** [comment]
Fixed by #301
